include/magma/vk/textures/TextureManager.h

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file TextureManager.h
+ * @brief Create and control textures
+ * @version 0.1
+ * @date 2021-02-12
+ */
+#pragma once
+#include <string>
+#include <unordered_map>
+
+#include "magma/vk/commandBuffer.h"
+#include "magma/vk/textures/Texture.h"
+
+class LogicalDevice;
+
+class TextureManager {
+public:
+    TextureManager(LogicalDevice &device);
+    ~TextureManager();
+
+    bool textureExist(const std::string &name) const;
+    Texture &getTexture(const std::string &name);
+
+    Texture &createTexture2D(const std::string &name, VkFormat format, VkExtent3D extent,
```

Comment:
I guess extent type should be VkExtent2D, because 2D texture doesn't have 3rd dimension.

Reply:
Done